ios/NewExpensify/Info.plist

```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.2.42</string>
+	<string>1.2.41</string>
```

Comment:
Why do we need to change this? This update will go out with the latest version, no?

Reply:
Oops jinx you fixed this!

Reply:
Xcode auto "corrected" this. I asked in Slack and reverted this. Thanks for the review and merge!